heapster-saw/src/Verifier/SAW/Heapster/SAWTranslation.hs

```diff
@@ -3806,16 +3806,18 @@ debugPrettyPermCtx prxs (ptranss :>: ptrans) =
 translateApply :: String -> OpenTerm -> Mb ctx (DistPerms ps) ->
                   ImpTransM ext blocks tops ret ps ctx OpenTerm
 translateApply nm f perms =
-  do expr_ctx <- itiExprCtx <$> ask
+  do assertPermStackEqM nm perms
```

Comment:
This addition seems unrelated to the purpose of the PR, just wanted to make sure it was intentional.